### PR TITLE
Feature/local development

### PR DESCRIPTION
**Pull request by zambrovski**
**Description**

- Local keycloak + init
- Local kafka init
- Gateway improvements in yml files to use local-docker.env eventually

### CODE REVIEW

File: digiwf-gateway/src/main/resources/application-local.yml [hunk 1]
```diff
@@ -18,25 +12,29 @@ spring:
...
           uri: http://localhost:39146/
           predicates:
             - Path=/api/digitalwf-backend-service/rest/**
           filters:
             - RewritePath=/api/digitalwf-backend-service/(?<urlsegments>.*), /$\{urlsegments}
             - RemoveResponseHeader=WWW-Authenticate
+
         - id: frontend
           uri: http://localhost:8081/
           predicates:
```
Comment (lmoesle) [inline digiwf-gateway/src/main/resources/application-local.yml]: If we use for the frontend and backend uri env variables we could remove the local and local-docker profile

File: stack/docker-compose.yml [hunk 2]
```diff
@@ -23,6 +23,33 @@ services:
...
+      echo -e 'Creating kafka topics...'
+      kafka-topics --bootstrap-server kafka:9092 --create --if-not-exists --topic $${KAFKA_TOPIC_TASKS} --replication-factor 1 --partitions 1
+      kafka-topics --bootstrap-server kafka:9092 --create --if-not-exists --topic $${KAFKA_TOPIC_DATA_ENTRIES} --replication-factor 1 --partitions 1
+
+      kafka-topics --bootstrap-server kafka:9092 --create --if-not-exists --topic $${KAFKA_TOPIC_COCREATION_DEPLOY} --replication-factor 1 --partitions 1
+      kafka-topics --bootstrap-server kafka:9092 --create --if-not-exists --topic $${KAFKA_TOPIC_COCREATION} --replication-factor 1 --partitions 1
+
+      kafka-topics --bootstrap-server kafka:9092 --create --if-not-exists --topic $${KAFKA_TOPIC_ENGINE} --replication-factor 1 --partitions 1
+      kafka-topics --bootstrap-server kafka:9092 --create --if-not-exists --topic $${KAFKA_TOPIC_ENGINE_DLQ} --replication-factor 1 --partitions 1
+
```
Comment (lmoesle) [inline stack/docker-compose.yml:53]: The topics for the integrations are missing. I will add them later

File: stack/local-docker.env [hunk 3]
```diff
@@ -34,3 +38,18 @@ MAIL_PASSWORD=secret
 KAFKA_BOOTSTRAP_SERVER=localhost
 KAFKA_BOOTSTRAP_SERVER_PORT=29092
 KAFKA_SECURITY_PROTOCOL=PLAINTEXT
+
+KAFKA_TOPIC_TASKS=dfw-taskmanagement-tasks-local-01
+KAFKA_TOPIC_DATA_ENTRIES=dfw-taskmanagement-data-local-01
+KAFKA_TOPIC_COCREATION_DEPLOY=dwf-cocreation-deploy-local-01
+KAFKA_TOPIC_COCREATION=dwf-cocreation-local-01
+KAFKA_TOPIC_ENGINE=dwf-digiwf-engine-local-01
+KAFKA_TOPIC_ENGINE_DLQ=dwf-digiwf-local-01-dlq
```
Comment (lmoesle) [inline stack/local-docker.env:50]: A few are missing. I will add them later

**Review by lmoesle — APPROVED** [review]

**Comment by zambrovski** [comment]
@StephanStrehlerCGI what needs to be done here? When can we merge it?

**Pull request merged**